package.json

```diff
@@ -46,42 +46,24 @@
       {
         "title": "Snyk Vulnerability Scanner",
         "properties": {
-          "snyk.advancedMode": {
-            "type": "boolean",
-            "default": false,
-            "description": "Toggle advanced tools for expert users",
-            "scope": "application"
-          },
           "snyk.token": {
             "type": "string",
             "default": "",
             "description": "API key",
             "scope": "application"
           },
-          "snyk.uploadApproved": {
-            "type": "boolean",
-            "default": false,
-            "markdownDescription": "User consent for code upload to Snyk in agreement with our [terms and conditions](https://snyk.io/policies/terms-of-service/?utm_source=vsc). **This setting will be removed soon. Please use `snyk.codeEnabled` instead.**",
-            "scope": "window"
-          },
```

Comment:
@michelkaporin, we are confident with removing uploadApproved parameter, right?

Reply:
@pavel-snyk yes